src/odin/enhancedtrippath.cc

```diff
@@ -1298,6 +1430,72 @@ uint32_t EnhancedTripLeg_Node::GetStraightestIntersectingEdgeTurnDegree(uint32_t
   return staightest_turn_degree;
 }
 
+uint32_t EnhancedTripLeg_Node::GetRightMostTurnDegree(uint32_t turn_degree,
```

Comment:
will replace constants with this [issue](https://github.com/valhalla/valhalla/issues/1739)